lib/auth/streamingV4/trailingChecksumTransform.js

```diff
@@ -0,0 +1,98 @@
+const { Transform } = require('stream');
+const { errors } = require('arsenal');
+
+/**
+ * This class is designed to handle the chunks sent in a streaming
+ * v4 Auth request
+ */
```

Comment:
Worth adding a comment to explain that we are not validating the checksums, and why